plugins/filter/quote.yml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2022 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION:
+  name: quote
+  author:
+  - Jordan Borean (@jborean93)
+  short_description: Quotes argument(s) for various Windows shells
+  description:
```

Comment:
```suggestion
    seealso:
    - module: ansible.windows.win_command
    - module: ansible.windows.win_shell
    description:
```

Reply:
Is this indentation correct, seems like it needs to be back 2 spaces.

Reply:
I'd double check it in a real editor for sure... I thought the same thing in the preview, went to edit it, and then in edit I counted spaces and it seemed correct, so I didn't see how I could fix it.

I've seen the online suggestion thing screw this up before...

Reply:
actually, this was probably my screw-up, I just edited it back to 2 spaces